packages/datadog-plugin-amqplib/src/client.js

```diff
@@ -7,15 +7,17 @@ const { getResourceName } = require('./util')
 
 class AmqplibClientPlugin extends ClientPlugin {
   static get id () { return 'amqplib' }
+  static get type () { return 'messaging' }
+  static get ioDirection () { return 'controlPlane' }
```

Comment:
Not sure I understand this, that definitely doesn't sound like a direction?

Reply:
Also, is this actually used somewhere?

Reply:
Agreed, this `ioDirection` should probably be more generically named. `subType` sounds a bit too generic to me, WDYT?

Of all 3 AMQP integrations, this one is special because it's auto-instrumented based on a `defs.js` file generated by the library, which includes control-plane calls for managing exchanges and queues. These get their own `amqp.command` operation name, which has to go somewhere that's neither `inbound` nor `outbound` in the naming schema.

Reply:
The fact that it's a control plane then can be inferred by `type=worker` + `kind=client`.

Reply:
Done in `f37411e4f`. Client integrations will get updated, so I'll generalize this part to client integrations when they're all updated.